### ISSUE TITLE
Is there a formatting setting for the `new` operator?

### ISSUE THREAD

**Issue by tillig** [OmniSharp/omnisharp-roslyn#2348]
```c#
private Dictionary<string, string> value = new ()
```

In this situation , the `new ()` part - space between `new` and `()`. StyleCop SA1000 wants a space there, but I can't figure out the .editorconfig or OmniSharp setting to make code formatting add that. Formatting always seems to _remove_ the space so it's `new()`. My Google-fu is failing me in finding the right setting, if there is one.

**Comment by tillig** [comment]
After fully quitting VS Code, doing a `git clean -dfx`, and restarting, the analyzer now seems to _not_ want a space there, which actually matches up with the formatting. Unclear what got messed up, but at least it's being consistent now.

I'd still be curious to find out the answer to what setting controls this, in case I get into trouble in the future.

**Comment by Gkjsdll** [comment]
I'm running into this problem & `git clean -dfx` did _not_ fix it for me.

#### omnisharp.json
I also tried setting every `Space*` and `Spacing*` rule to true, and no space was added in `new()`.
```
{
  "FormattingOptions": {
    "IndentationSize": 2,
    "TabSize": 2,
    "UseTabs": false
  },
  "RoslynExtensionsOptions": {
    "EnableAnalyzersSupport": true,
    "LocationPaths": [
      "./Assets/Packages/StyleCop.Analyzers.1.1.118/",
      "./Assets/Packages/Microsoft.Unity.Analyzers.1.14.0/"
    ]
  }
}
```

#### stylecop.json
```
{
  "$schema": "https://raw.githubusercontent.com/DotNetAnalyzers/StyleCopAnalyzers/master/StyleCop.Analyzers/StyleCop.Analyzers/Settings/stylecop.schema.json",
  "settings": {
    "indentation": {
      "indentationSize": 2,
      "tabSize": 2,
      "useTabs": false
    },
    "layoutRules": {
      "newlineAtEndOfFile": "require"
    }
  }
}
```

#### Versions
- StyleCop.Analyzers - 1.1.118
- omnisharp - 1.39.0

**Comment by 2BitSalute** [comment]
Yeah, I'm having the same issue. I don't believe there is a setting for this, so I'm forced to change the code to explicitly name the types.

**Comment by Youssef1313** [comment]
SA1000 was already fixed. See https://github.com/DotNetAnalyzers/StyleCopAnalyzers/issues/3561
